common/special_projects/hfe_project.txt

```diff
@@ -434,9 +435,10 @@ special_project = {
 	picture = GFX_evt_hfe_thoth
 	icon = gfx/interface/icons/situation_log/situation_log_debris.dds
 	requirements = {
-		shipclass_science_ship = 1
-		leader = scientist
-		skill > 2
+		shipclass_constructor = 1
```

Comment:
Its intended to be science ship

Reply:
That makes absolutely no sense. Thoth is a huge, probably gigantic space station. A simple space walk  cannot install enough memory on Thoth to work.